bindings/python/include/ADIOSPy.h

```diff
@@ -23,7 +23,7 @@
 #include "adios2/pybind11/pybind11.h"
 #endif
 
-#include "adios2/ADIOS.h"
+#include "../../../source/adios2/core/ADIOS.h"
```

Comment:
@chuckatkins Thanks! I haven't worked on the Python bindings. I am actually working on it now.